organization_invitations.go

```diff
@@ -0,0 +1,62 @@
+// Package aiven provides a client for using the Aiven API.
+package aiven
+
+type (
+	// OrganizationsInvitationsHandler is the client which interacts with the Organization Invitations API on Aiven.
+	OrganizationInvitationsHandler struct {
```

Comment:
OrganizationInvitationsHandler
OrganizationUserInvitationsList

How about naming this same way: either OrganizationInvitations or OrganizationUserInvitations?